samples/hello-oboe/src/main/java/com/google/sample/oboe/hellooboe/MainActivity.java

```diff
@@ -81,18 +81,34 @@ public boolean onTouchEvent(MotionEvent event) {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-
+        mLatencyText = findViewById(R.id.latencyText);
         setupAudioApiSpinner();
         setupPlaybackDeviceSpinner();
         setupChannelCountSpinner();
         setupBufferSizeSpinner();
 
-        // initialize native audio system
+    }
+    /*
+    * Creating engine in onResume() and destroying in onPause() so the stream retains exclusive
+    * mode only while in focus. This allows other apps to reclaim exclusive stream mode.
+    */
+    @Override
+    protected void onResume() {
+        super.onResume();
         PlaybackEngine.create(this);
-
-        // Periodically update the UI with the output stream latency
-        mLatencyText = findViewById(R.id.latencyText);
         setupLatencyUpdater();
+        // Return the spinner states to their default value
+        mChannelCountSpinner.setSelection(1);
```

Comment:
Nit: It'd be clearer to use named constants here e.g. `private final int kDefaultChannelCountSelection = 1`